### PR TITLE
Team JFB website

### PR DESCRIPTION
**Pull request by lisazhao513**

### CODE REVIEW

File: content/projects/2022-implicit/index.md [hunk 1]
```diff
@@ -1,10 +1,103 @@
...
+## Solving Inverse Problems from a Classical Approach
+Using direct inverse we have:
+$$ d = \mathcal{A} x^\ast + \varepsilon \Longrightarrow x^\ast =  \mathcal{A}^{-1} d - \mathcal{A}^{-1} \varepsilon $$
+However, since $\varepsilon$ is unknown, directly inverting may end up amplifying this noise factor⁠. Because of this noise corruption, the reconstructed image ends up being unrecognizable. 
+
+To better visulaize this, we have the following set of pictures: 
+
+![Original Image](imgs/inverse1.png "Original Image") ![Blurred Noisy Image](imgs/inverse2.png "Blurred Noisy Image") ![Direct Inverse](imgs/inverse3.png "Direct Inverse")
+
+In order to minimize the noise factor, we want to formulate an optimization problem.
```
Comment (lruthotto) [inline content/projects/2022-implicit/index.md:31]: I'd argue that you want to formulate a *regularized* optimization problem.

File: content/projects/2022-implicit/index.md [hunk 2]
```diff
@@ -1,10 +1,103 @@
...
+![Original Image](imgs/inverse1.png "Original Image") ![Blurred Noisy Image](imgs/inverse2.png "Blurred Noisy Image") ![Gradient Descent](imgs/gd4.png "Gradient Descent")
+
+We see that the reconstructed image using gradient descent is a huge improvement from direct inverse; however, there are still blurry areas we can improve on. In search of a better method, we turn towards implicit learning.  
+
+## Implicit Deep Learning
+The issue with the classical approach is that the regularizer is chosen heristically. To combat this, our approach now is to utilize data to **learn** and **train** the regularizer.
+
+To do this we mimic gradient descent, but replace the gradient of the regularizer, $\lambda \nabla_x R$, with a trainable network. 
+
+However, this creates some problems concerning memory cost and the number of layers, $K$, in our neural network. The memory grows linearly as $K$—chosen heuristically—increases.
```
[the remark this answers is not in the document]
Comment (lruthotto) [reply content/projects/2022-implicit/index.md:48]: Also, is this point obvious or do you want to reference a paper where I can see why this is the case?

File: content/projects/2022-implicit/index.md [hunk 3]
```diff
@@ -1,10 +1,103 @@
...
+## Implicit Backpropagation 
+Suppose now we have found a fixed point $x^\ast$ for a single layer. Then, $$ x^\ast = T_\Theta (x^\ast) $$
+
+Using implicit differentiation on the equation above we have, 
+
+$$\frac{d x^\ast}{d \Theta} = \left( I - \frac{d T_\Theta (x^\ast)}{d x^\ast}\right)^{-1} \frac{\partial T_\Theta (x^\ast)}{\partial \Theta}$$ 
+
+However, solving this is very expensive because of the inverse term.
+
+To circumvent this issue, we use a recently proposed method called Jacobian-Free Backpropagation.
```
Comment (lruthotto) [inline content/projects/2022-implicit/index.md:61]: link the paper?

File: content/projects/2022-implicit/index.md [hunk 4]
```diff
@@ -1,10 +1,103 @@
...
+
+## Results
+The results are as follows
+![loss plot](imgs/loss_plot-1.jpg)
+From the graph we see that the loss is decreasing as the number of epochs increases. An epoch is one complete pass of the entire dataset through our algorithm. 
+![results](imgs/truth_36-1.jpg)
+Note: Two metrics are commonly used for assessing the quality of reconstructed images: the peak-signal-to-noise ratio (PSNR, a positive number, best at $+\infty$) and the structural similarity index measure (SSIM, also positive, best at $1$).
+
+## Acknowledgements
+We sincerely thank the guidance of our mentor, Dr. Samy Wu Fung, and other mentors at Emory University for the opportunity. This work is supported in part by the US National Science Foundation awards DMS-2051019 and DMS-1751636.
```
Comment (lruthotto) [inline content/projects/2022-implicit/index.md:96]: No need to list the grants here. They are listed on our main website.  OK of course to leave the first sentence if you like.

**Review by lruthotto — CHANGES_REQUESTED** [review]
Good job, team. This looks really nice, but I left a few easy-to-address comments for you to fix. Please write me on slack if you have questions about any of these.

File: content/projects/2022-implicit/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (lliu58b) [reply content/projects/2022-implicit/index.md:48]: capitalized K is the number of layers. small k is the iteration index. This is indeed a bit confusing. should we change it to another index like j? 

This point is from this IEEE paper <https://ieeexplore.ieee.org/document/9565378>, on page 1124, section II.A.

[the remark this answers is not in the document]
Comment (lruthotto) [reply content/projects/2022-implicit/index.md:48]: Now that K is defined this is better. Do you want to link the paper?

**Comment by lruthotto** [comment]
Looks good to me now! Thanks a lot.

**Pull request merged**